.github/workflows/check.yml

```diff
@@ -0,0 +1,57 @@
+name: check
+
+on:
+  push:
+    branches:
+      - es2019
```

Comment:
does this limit CI to just the `es2019` branch?

Reply:
All pull requests and then also the es2019 branch, yeah.